main.js

```diff
@@ -3001,7 +3001,7 @@ function startScheduledInstance(callback) {
                 states.setState(instance._id + '.sigKill', {val: 0, ack: false, from: hostObjectPrefix}, () => {
                     const args = [instance._id.split('.').pop(), instance.common.loglevel || 'info'];
                     try {
-                        procs[id].process = cp.fork(fileNameFull, args, {windowsHide: true});
```

Comment:
Why is windowsHide removed?

Reply:
fork has no option windowsHide, see also https://github.com/ioBroker/ioBroker.js-controller/issues/1036

Reply:
Yes, it has (some kind of undocumented). 
Otherwise all windows user will get for every adapter a "cmd.exe" window visible.
Looks not so good. I added this flag myself and after that all "cmd.exe" was invisible if started from CLI and not as service.

Reply:
But actually I can find only windowsHide for exec(Sync), spawn(Sync) and execFile(Sync)...